security_monkey/manage.py

```diff
@@ -636,17 +636,17 @@ def sync_swag(owner, bucket_name, bucket_prefix, bucket_region, account_type, sp
     account_manager = account_registry[account_type]()
 
     for account in swag.get_all("[?provider=='{provider}']".format(provider=account_type.lower())):
-        active = False
         services = account.get('services', [])
-        for s in services:
-            if s['name'] == 'security_monkey':
-                for status in s['status']:
-                    if status['region'] == 'all':
-                        active = status['enabled']
+        services_by_name = {s['name']: s for s in services}
 
-        thirdparty = True
-        if account['owner'] == owner:
-            thirdparty = False
+        secmonkey_service = services_by_name.get('security_monkey', {})
+        all_region_status = {}
+        for status in secmonkey_service.get('status', []):
```

Comment:
I'm wondering if all of this code should be wrapped in:
```
if securitymonkey_service:
   ...
```

Reply:
Up to you, honestly! I have no strong opinion.

My personal preference is this flatter `settings.get(key, default)` pattern. IMO it's less mental load than backtracking through conditionals.